net/src/ipc_net_worker.rs

```diff
@@ -108,10 +108,7 @@ impl IpcNetWorker {
         if config["socketType"] != "zmq" {
             bail!("unexpected socketType: {}", config["socketType"]);
         }
-        let mut block_connect = true;
-        if let Some(b) = config["blockConnect"].as_bool() {
-            block_connect = b;
-        }
+        let block_connect = config["blockConnect"].as_bool().unwrap_or(true);
         if let None = config["ipcUri"].as_str() {
```

Comment:
Please use
```rust
if config["ipcUri"].as_str().is_none() {
```
here

Reply:
Why?

This should compile into equivalently efficient optimized byte code, correct? The terseness is equivalent and the readability to my eyes only differs subjectively.

Reply:
It's just a common way of writing this in Rust. Also clippy would argue about this 😉